### PR TITLE
clean tmp file for SaveMultipartFile

### PR DESCRIPTION
**Pull request by laoshanxi**
To fix issue:
https://github.com/valyala/fasthttp/issues/1439

### CODE REVIEW

File: brotli.go [hunk 1]
```diff
@@ -8,7 +8,7 @@ import (
 
 	"github.com/andybalholm/brotli"
 	"github.com/valyala/bytebufferpool"
-	"github.com/valyala/fasthttp/stackless"
+	"github.com/laoshanxi/fasthttp/stackless"
```
Comment (li-jin-gou) [inline brotli.go:11]: Dependence is changed by you, you should not use your own.

**Review by li-jin-gou — CHANGES_REQUESTED** [review]

File: brotli.go (hunk 1 above)
Comment (laoshanxi) [reply brotli.go]: thanks for remind, the second commit automatic come to the pull request, now it's fine.

Comment (erikdubbelboer) [reply brotli.go]: A good tip is to never make a pull request from master. Always create a new branch to use for the pull request.

Comment (laoshanxi) [reply brotli.go]: > A good tip is to never make a pull request from master. Always create a new branch to use for the pull request.

You're right, Thanks!

**Pull request closed without merging**